releasenotes/notes/removed-ibmq-provider-1c757ce5ef01fb19.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    ``qiskit-ibmq-provider`` is deprecated and has been dropped as a requirement. Interactions
+    with backends should use the ``qiskit-ibm-provider`` package instead (must be installed
```

You might have missed this part of my previous suggestion:

```suggestion
    with IBM backends should use the ``qiskit-ibm-provider`` package instead (must be installed
```